packages/e2e/features/ui/components/authenticator/verify-user.feature

```diff
@@ -7,14 +7,15 @@ Feature: Verify User
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-with-email"
 
-  @angular @next @vue
+  # Verify not functioning in Angular
+  @react @vue
```

Comment:
@wlee221 Based on #369 this should've been

```suggestion
  @angular @react @vue
```

Reply:
Let's go for it!